testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/SinglenodeCompatibility.java

```diff
@@ -99,6 +101,21 @@ private String getConfigurationDirectory(String dockerImageName)
         throw new RuntimeException("Unexpected docker image " + dockerImageName);
     }
 
+    private void configureTestsContainer(Environment.Builder builder, Config config)
+    {
+        String temptoConfig;
+        if (config.getCompatibilityTestDockerImage().startsWith("ghcr.io/trinodb/presto")) {
```

Comment:
Elsewhere you are doing
```
DockerImageName.parse(dockerImageName).getRepository() equals "trinodb/presto"
```

however, i think it would be cleaner to just base it on `parseInt(versionString)`